frontend/src/layouts/Header/Header.tsx

```diff
@@ -1,15 +1,26 @@
 import './Header.css'
 import FlightTakeoffIcon from '@mui/icons-material/FlightTakeoff';
+import {Link} from "react-router-dom";
 
 export default function Header() {
 
     return(
         <header>
             <FlightTakeoffIcon
-                sx={{ fontSize: "25px"}}
-                onClick={() => window.location.href='/'}
-                style={{ marginLeft: "20px", cursor: "pointer"}}
+                sx={{fontSize: "35px"}}
```

Comment:
I woud recommend not to or to avoid use of fix values for size... if u use units like % or rem, the design will responsive!